psake.psm1

```diff
@@ -574,11 +574,11 @@ function ConfigureBuildEnvironment {
         }
         {($_ -eq '4.5.1') -or ($_ -eq '4.5.2')} {
             $versions = @('v4.0.30319')
-            $buildToolsVersions = @('14.0', '12.0')
+            $buildToolsVersions = @('15.0', '14.0', '12.0')
         }
         {($_ -eq '4.6') -or ($_ -eq '4.6.1')} {
             $versions = @('v4.0.30319')
-            $buildToolsVersions = @('14.0')
+            $buildToolsVersions = @('15.0', '14.0')
         }
 

```

Comment:
Just added these frameworks.  I meant to do it when it was first suggested but I went on vacation and forgot.